### PR TITLE
Mention the Playground in the `README.md`

### PR DESCRIPTION
**Pull request by ggreif**
Also adding a link to the IDE. Plus a lot of other things.

### CODE REVIEW

**Comment by ggreif** [comment]
I imagine, we can add more links here.

File: README.md [hunk 1]
```diff
@@ -46,5 +46,7 @@ A safe, simple, actor-based programming language for authoring [Internet Compute
 
 ## Related repositories
 
-* [Base library](https://github.com/dfinity/motoko-base).
-* [Vessel package manager](https://github.com/dfinity/vessel).
+* [Base library](https://github.com/dfinity/motoko-base)
+* [Vessel package manager](https://github.com/dfinity/vessel)
+* [Motoko Playground](https://github.com/dfinity/motoko-playground)
+  * [Live IDE](https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app)
```
Comment (rvanasa) [inline README.md:52]:
```suggestion
* [Example projects](https://github.com/dfinity/examples/tree/master/motoko)
* [Motoko Playground](https://github.com/dfinity/motoko-playground) &middot; ([online IDE](https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app))
* [Embed Motoko code snippets](https://github.com/dfinity/embed-motoko) &middot; ([online interpreter](https://embed.smartcontracts.org/))
* [VS Code extension](https://github.com/dfinity/vscode-motoko) &middot; ([install](https://marketplace.visualstudio.com/items?itemName=dfinity-foundation.vscode-motoko))
* [Browser and Node.js bindings](https://github.com/dfinity/node-motoko) &middot; ([npm package](https://www.npmjs.com/package/motoko))
```

* [Example projects](https://github.com/dfinity/examples/tree/master/motoko)
* [Motoko Playground](https://github.com/dfinity/motoko-playground) &middot; ([online IDE](https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app))
* [Embed Motoko code snippets](https://github.com/dfinity/embed-motoko) &middot; ([online interpreter](https://embed.smartcontracts.org/))
* [VS Code extension](https://github.com/dfinity/vscode-motoko) &middot; ([install](https://marketplace.visualstudio.com/items?itemName=dfinity-foundation.vscode-motoko))
* [Browser and Node.js bindings](https://github.com/dfinity/node-motoko) &middot; ([npm package](https://www.npmjs.com/package/motoko))

Comment (rvanasa) [reply README.md:52]: Maybe we could include some community resources in another section:

- [Awesome Motoko](https://github.com/motoko-unofficial/awesome-motoko#readme)
- [Motoko Bootcamp](https://github.com/motoko-bootcamp/bootcamp#readme) &middot; ([YouTube channel](https://www.youtube.com/channel/UCa7_xHjvOESf9v281VU4qVw))
- [NFT](https://github.com/rocklabs-io/ic-nft) and [token](https://github.com/rocklabs-io/ic-token) implementations in Motoko

Comment (crusso) [reply README.md:52]: I would maybe shy away from the NFT links, at east until there's some well-reviewed standard. Others seem good.

Comment (rvanasa) [reply README.md:52]: Sounds good to me. While people are definitely looking for NFT / token implementations (at least within the Blocks user base), the "Awesome Motoko" link should be sufficient for holding these implementations at arm's length until there's an official standard.

**Pull request merged**